### PR TITLE
Fixed incorrectly handled image upload errors

### PR DESCRIPTION
**Pull request by kamilpastuszka**
I want to merge this change to provide users with proper error notifications if they try to upload image files in a wrong format.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

File: src/intl.ts [hunk 1]
```diff
@@ -49,6 +49,13 @@ export const commonMessages = defineMessages({
   generalInformations: {
     defaultMessage: "General Information"
   },
+  imgageUploadError_title: {
+    defaultMessage: "Couldn't process image"
+  },
+  imageUploadError_text: {
+    defaultMessage:
+      "There was a poblem with the file you uploaded as an image and it couldn't be used. Please try a different file."
+  },
```
Comment (dominik-zeglen) [inline src/intl.ts:58]: I don't think these are common messages. They should be defined somewhere around the component using them.

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

File: src/intl.ts (hunk 1 above)
Comment (kamilpastuszka) [reply src/intl.ts:58]: The content has been confirmed internally, and is included in the ticket.

**Review by orzechdev — CHANGES_REQUESTED** [review]
Error still causes displaying an old general message in product and collection pages and nothing is displayed during user avatar update.

**Comment by kamilpastuszka** [comment]
It’s been fixed now for product and collection pages as well as user avatar update. 
Previous changes based on initial scope.

File: src/intl.ts [hunk 2]
```diff
@@ -49,6 +49,13 @@ export const commonMessages = defineMessages({
   generalInformations: {
     defaultMessage: "General Information"
   },
+  imgageUploadError_title: {
```
Comment (orzechdev) [inline src/intl.ts:52]: These names should also be camelCase to make it consistent with the rest naming convention. I don't see a reason why they aren't.

**Review by orzechdev — CHANGES_REQUESTED** [review]

File: src/intl.ts (hunk 1 above)
Comment (dominik-zeglen) [reply src/intl.ts:58]: `errorMessages` sounds good

File: src/intl.ts (hunk 2 above)
Comment (kamilpastuszka) [reply src/intl.ts]: Fixed.

File: src/intl.ts (hunk 1 above)
Comment (kamilpastuszka) [reply src/intl.ts]: Changed for the errors I dealt with. Probably a good idea to adjust for other types of errors still in commonMessages.

**Review by dominik-zeglen — APPROVED** [review]

**Review by orzechdev — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**